index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+
+<head>
+	<title>Ecxercise 3</title>
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,400&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <nav class="nav-bar"> <!-- Flex parent -->
+
+        <div class="nav-menu"> <!-- left menu -->
+            <div class="nav-menu-item"> <!-- Home button -->
+                <a href="#home">home</a>
+
+            </div>
+
+            <div class="nav-menu-item"> <!-- email button-->
+                <a href="mailto:newton.maxwell@gmail.com">email</a>
+            </div>
+        </div>
+
+        <div class="nav-title"> <!-- center heading -->
+            <p class=site-title>maxwell newton</p>
+        </div>
+
+        <div class="nav-menu"> <!-- right menu-->
+
+            <div class="nav-menu-item"> <!-- LinkedIn button -->
+                <a href="https://www.linkedin.com/in/maxwellnewton/">linkedin</a>
+            </div>
+
+            <div class="nav-menu-item"> <!-- GitHub button -->
+                <a href="https://github.com/maxwell-01">github</a>
```

Comment:
for these elements, you want the content to be capitalised like so:

Home
Email
Maxwell Newton
LinkedIn
GitHub

this is because if you decide to change the styling in the future to not be all caps, that way they will be spelled correctly